IORequestGenerator/IORequestGenerator.cpp

```diff
@@ -1734,7 +1734,7 @@ DWORD IORequestGenerator::_CreateDirectoryPath(const char *pszPath) const
 //
 bool IORequestGenerator::_CreateFile(UINT64 ullFileSize, const char *pszFilename, bool fZeroBuffers, bool fVerbose) const
 {
-    bool fSlowWrites = false;
+    bool fSlowWrites = true;
```

Comment:
why slow writes is default mode?

Reply:
Below code block setting fSlowWrites =true if SetPrivilige fails also became wrong since fSlowWrites would be true there anyway